web/src/compose_closed_ui.js

```diff
@@ -101,11 +101,21 @@ function update_new_direct_message_button(btn_text) {
     $("#new_direct_message_button").text(btn_text);
 }
 
+function hide_new_direct_message_button(is_direct_message_narrow) {
+    const $new_direct_message_button_container = $(".new_direct_message_button_container");
+    if (is_direct_message_narrow) {
+        $new_direct_message_button_container.hide();
+    } else {
+        $new_direct_message_button_container.show();
+    }
+}
```

Comment:
I'm not thrilled with a function being called `hide_*` that can sometimes show things.

Reply:
No, for sure. `toggle_visiblity_of` or something would be better.